spec/components/anchor/table_component_spec.rb

```diff
@@ -149,4 +149,67 @@ def [](key)
 
     expect(page).to have_css "caption", text: "A caption", class: "sr-only"
   end
+
+  describe "accepts a `total` attribute to display the number of rows" do
+    context "with no argument" do
+      it "renders no total" do
+        component = described_class.new(data: [])
+
+        render_inline(component) do |table|
+          table.with_column(header: :id, value: :id)
+          table.with_column(header: :name, value: :name)
+        end
+
+        expect(page).not_to have_text "Total: 0"
```

Comment:
I'd remove the zero so we know it doesn't appear regardless of spec setup (no need to double check):
```suggestion
        expect(page).not_to have_text "Total:"
```